ios/Midnight.m

```diff
@@ -1,19 +1,14 @@
-#import "Midnight.h"
-
-@implementation Midnight
-
-RCT_EXPORT_MODULE()
-
-// Example method
-// See // https://reactnative.dev/docs/native-modules-ios
-RCT_REMAP_METHOD(multiply,
-                 multiplyWithA:(nonnull NSNumber*)a withB:(nonnull NSNumber*)b
-                 withResolver:(RCTPromiseResolveBlock)resolve
-                 withRejecter:(RCTPromiseRejectBlock)reject)
-{
-  NSNumber *result = @([a floatValue] * [b floatValue]);
-
-  resolve(result);
-}
+#import <React/RCTBridgeModule.h>
+#import <React/RCTEventEmitter.h>
+
+@interface RCT_EXTERN_MODULE(Midnight, NSObject)
+RCT_EXTERN_METHOD(multiply:(nonnull double*)a
```

Comment:
Can we remove all instances/references to multiply? We shouldn't need that anymore 😄